README.adoc

```diff
@@ -342,6 +342,7 @@ leakHunter: false
 mergeDataSets: false
 caseInsensitiveStrategy: !!com.github.database.rider.core.api.configuration.Orthography 'UPPERCASE' <1>
 raiseExceptionOnCleanUp: false <2>
+expectedDbType: HSQLDB <3>
```

Comment:
I think here the default is UNKNOWN, right? is this doc we should put the default values

Reply:
OK, I modify it to UNKNOWN.